src/main/groovy/com/github/jrubygradle/jar/JRubyJarConfigurator.groovy

```diff
@@ -24,15 +24,6 @@ class JRubyJarConfigurator {
         configure()
     }
 
-    @PackageScope
```

Comment:
I'm not sure whether I am happy with dropping shadowJar support completely. What I do think is to definitely remove it from a fixed dependency in `build.gradle`. It might still be worthwhile for someone who want to use that instead to go that way bu explicitly including it in their buildscript and applying the plugin.